src/containers/VariationsMainView/VariationsMainView.js

```diff
@@ -105,10 +131,25 @@ class VariationsMainView extends React.Component {
               onVolumeChanged={this.props.setVolume}
               onPlay={this.props.play}
               onPause={this.props.pause}
-              onNextBubble={() => {}}
-              onPreviousBubble={() => {}}
+              onNextBubble={this.props.nextBubble}
+              onPreviousBubble={this.props.previousBubble}
               onScrubAhead={this.props.fastForward}
               onScrubBackwards={this.props.fastReward}
+              onAddBubble={
+                selectedBubbles.length === 1
+                  ? this.addRange(selectedBubbles[0])
+                  : undefined
```

Comment:
`null` is generally preferred over `undefined`. `undefined` can be used to catch places in code you didn't expect there not to be a value (possible bugs).

Reply:
https://github.com/digirati-co-uk/timeliner/pull/105/commits/1fbd88e8d18fa550fb06fd2cdb6caa8d8905e033